src/contracts/core/AVSDirectory.sol

```diff
@@ -451,6 +526,26 @@ contract AVSDirectory is
         }
     }
 
+    /**
+     * @notice Returns an array of strategies in the operatorSet.
+     * @param operatorSet The operatorSet to query.
+     * @param start The starting index of the array to query.
+     * @param length The amount of items of the array to return.
+     */
+    function getStrategiesInOperatorSet(
```

Comment:
@bowenli86 would it make your life easier if we just return all the enumerated strategies in the operator set?  I don't love this pattern for paginating these values with start and length params